app/services/api/v1x0/catalog.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1x0
+    module Catalog
+
+    end
+  end
+end
```

Comment:
Do we have to have this open or can we do `module Api::V1x0::Catalog`?

Reply:
Yes. Getting them added.